daemon/graphdriver/aufs/aufs.go

```diff
@@ -125,7 +125,7 @@ func Init(root string, options []string, uidMaps, gidMaps []idtools.IDMap) (grap
 	// Create the root aufs driver dir and return
 	// if it already exists
 	// If not populate the dir structure
-	if err := idtools.MkdirAllAs(root, 0700, rootUID, rootGID); err != nil {
+	if err := idtools.MkdirAllAndChown(root, 0700, idtools.IDPair{UID: rootUID, GID: rootGID}); err != nil {
```

Comment:
Perhaps for the sake of readability/simplicity it makes sense to do something like

```go
owner :=  idtools.IDPair{UID: rootUID, GID: rootGID}
```
and then use it 
```go
if err := idtools.MkdirAllAndChown(root, 0700, owner); err != nil {
```

otherwise it looks like excessive typing. Same for all similar places.

Reply:
See my earlier comment; re: using a variable or not https://github.com/moby/moby/pull/35541#discussion_r152264012